ethcore/src/engines/authority_round/mod.rs

```diff
@@ -57,14 +58,19 @@ use unexpected::{Mismatch, OutOfBounds};
 
 mod finality;
 
+type TransitionStep = u64;
+type TransitionTimestamp = u64;
+type StepDuration = u64;
```

Comment:
was it changed to `u64` to avoid casts?

Reply:
We have triples of `u64` which needed some explanation in https://github.com/paritytech/parity-ethereum/blob/7863f8381a3790a3e62c605b6fc27c6c6b7fc5fe/ethcore/src/engines/authority_round/mod.rs#L157.

@ordian Would you prefer a struct type instead of plain triples?

Reply:
@vkomenda not sure if the fields would be named like `transition_step, transition_timestamp, step_duration`, it would be any clearer, but I don't have a strong preference here.

Reply:
> @vkomenda not sure if the fields would be named like `transition_step, transition_timestamp, step_duration`, it would be any clearer, but I don't have a strong preference here.

That might actually help to avoid confusing different fields when using the triple. I'll add that struct.